localization/english/wc_council_tasks_l_english.yml

```diff
@@ -1,3 +1,4 @@
 ﻿l_english:
     #steward
-     STEWARD_PROMOTE_CULTURE_INNOVATION_BONUS_EPIDEMIC:1 "[councillor_liege.GetCulture.GetName] — Epidemic"
\ No newline at end of file
+     STEWARD_PROMOTE_CULTURE_INNOVATION_BONUS_EPIDEMIC:1 "[councillor_liege.GetCulture.GetName] — Epidemic"
```

Comment:
Easy and clear, puts the innovation name. Also, don't forget to change other languages.
```suggestion
     STEWARD_PROMOTE_CULTURE_INNOVATION_BONUS_EPIDEMIC:1 "[councillor_liege.GetCulture.GetName] — $innovation_epidemic$"
```

Reply:
I will keep this method in mind when making the tenant, but it wont be tagged $innovation_epidemic$, so I will just comment the line out for now